.gitmodules

```diff
@@ -7,7 +7,7 @@
 	url = https://github.com/glennrp/libpng
 [submodule "extlib/freetype"]
 	path = extlib/freetype
-	url = https://git.savannah.nongnu.org/r/freetype/freetype2.git/
+	url = https://git.savannah.nongnu.org/git/freetype/freetype2.git
```

Comment:
I did the same yesterday (locally to test).

Why did you change the URL? Both of them apparently point to the same git repository.

Reply:
Oooh please please keep the new one with the `git` in the url, that one supports shallow cloning (`--depth=1`) and the current one does not, causing a lot of pain when building in nix and otherwise.

Good find! 👌

Reply:
OK this makes sense.

Reply:
I'm glad it solves an issue, but TBH I just did it because the URL looked a bit weird to me with the slash at the end, and when I looked up the URL on the FreeType project page I couldn't find it, so I assumed it was perhaps an old repository and used the up to date URL instead.